ste/jobStatusManager.go

```diff
@@ -61,14 +61,28 @@ func (jm *jobMgr) statusMgrClosed() bool {
 
 /* These functions should not fail */
 func (jm *jobMgr) SendJobPartCreatedMsg(msg JobPartCreatedMsg) {
-	jm.jstm.partCreated <- msg
-	if msg.IsFinalPart {
-		// Inform statusManager that this is all parts we've
-		close(jm.jstm.partCreated)
+	defer func() {
```

Comment:
these fixes will technically resolve the issue, but have we looked into if this function is being called when it shouldnt be?

Reply:
Yes, I looked into that and this function is called only one other time. When we update the jobPart Status with the status Manager -- which seems to be fine to me!